charter.md

```diff
@@ -1,52 +1,124 @@
-is to be a live, responsive body that can refactor and reform as necessary 
-to adapt to a changing project and community.
+The Kubernetes Steering Committee is the governing body of the Kubernetes
+project, providing decision-making and oversight pertaining to the Kubernetes
+project bylaws, sub-organizations, and financial planning.  The Steering 
+Committee also defines the project values and structure.
 
-## Scope and Objectives
-The Steering Committee has a set of rights and responsibilities including the
-following:
+## How
 
-* Decide which project subgroups are responsible for each area of the project
-  and to delegate appropriate authority those groups.
-* Define, evolve, and defend the vision, values, mission, and scope of the
-  project.
+* Provide top-level accountability for the health of the project.
+* Delegate management<sup>[1](#footnote1)</sup> to other
+  parties<sup>[2](#footnote2)</sup>.
+* Adapt the role and structure of the Steering Committee as needed to meet the
+  needs of the project.
+* Responsibilities not explicitly delegated to other
+  parties<sup>[2](#footnote2)</sup> through their charters reside with
+  the Steering Committee.
+
+## Direct responsibilities of the Steering Committee
+
+The following responsibilities belong to the Steering Committee.
+
+* Act as escalation point of last resort for decisions important to the 
+  project that cannot be resolved within SIGs or Committees.
+* Delegate project responsibilities and ownership to SIGs and Committees.
+* Define, evolve, and defend the non-technical vision / mission and values
+  of the project.
+* Request funds and other support from the CNCF (e.g. marketing, press, etc.)
+* Define and enforce requirements for community groups<sup>[3](#footnote3)</sup>
+  to be in good standing such as having an approved charter.
+* Through Charters, decide which parties<sup>[2](#footnote2)</sup> are 
+  responsible for non-technical areas of the project.
+* Define and evolve guidance for group<sup>[3](#footnote3)</sup> governance
+  structures and policies, including how contributors become 
+  committers/maintainers, approvers, reviewers, members, etc.  As well as 
+  responsibilities associated with these roles.
 * Charter and refine policy for defining new community groups (including
   Special Interest Groups, Working Groups, and Committees), and establish
-  transparency and accountability policies for such groups
-* Define, evolve, and defend a Code of Conduct, which must include a neutral,
-  unbiased process for resolving conflicts
-* Define and evolve project governance structures and policies, including how
-  contributors become committers/maintainers, approvers, reviewers, members,
-  etc.
+  transparency and accountability policies for such groups.
+  
+### Not yet delegated responsibilities
+
+The following responsibilities may be delegated in the future.
+
+* Manage the Kubernetes brand to decide which things can be called “Kubernetes”
```

Comment:
We don't have ownership or enforcement here, that resides with the CNCF, but we can coordinate with them.

Reply:
I agree